emily/lambda/src/resources/deposits.rs

```diff
@@ -0,0 +1,487 @@
+use std::{collections::HashMap, convert::{From, TryFrom}, fmt::format, str::FromStr};
+
+use aws_sdk_dynamodb::types::AttributeValue;
+use emily::models::{self, DepositBasicInfo, OpStatus};
+
+use crate::errors::EmilyApiError;
+
+// ------------------------------------------------------------------
+// Api resources.
+// ------------------------------------------------------------------
+
+/// Deposit resource.
+#[derive(Clone)]
+pub struct DepositResource {
```

Comment:
I'd probably remove all of these and just use the table entry structs. They seem unnecessary.

Also, I'd maybe rename some of the table entry structs. For example, maybe `DepositTableEntry` would be `DepositRequest`? I don't know, you could also keep the names the same.